docs/admin/registration.rst

```diff
@@ -0,0 +1,41 @@
+Artemis supports user registration based on the Jhipster template.
+User registration has to be enabled in one ``application-*.yml`` file and can be customized.
+
+Example:
+
+.. code:: yaml
+
+    artemis:
+        user-management:
+            use-external: false
+            registration:
+                enabled: true
+                allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@(tum.de)|(in.tum.de)|(mytum.de)
```

Comment:
This line seems erroneous, as we don't expect a `.` in the domain, but allow any value.
This would e.g. allow the domain `inAtum.de` to register users.
I believe this can be fixed be escaping the dot.
```suggestion
                allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@(tum\.de)|(in\.tum\.de)|(mytum\.de)
```

Reply:
You can test it on https://regexr.com/ btw

Reply:
I think it needs to be `([a-zA-Z0-9_\-\.\+]+)@((tum\.de)|(in\.tum\.de)|(mytum\.de))` otherwise 
![Uploading image.png…]()

Reply:
fixed